js_modules/dagster-ui/packages/ui-core/src/instigation/TickDetailsDialog.tsx

```diff
@@ -175,9 +183,6 @@ const TickDetailsDialogImpl = ({tickId, tickResultType, instigationSelector}: In
           ) : null}
         </div>
       ) : null}
-      {activeTab === 'logs' ? (
-        <QueryfulTickLogsTable instigationSelector={instigationSelector} tick={tick} />
-      ) : null}
```

Comment:
I think this TickDetailsDialog is also shown when you click on the tick's name in the "Launched By" tag (`<TickTagForRun />`)

It might be nice to make that tag open the little black tooltip with "View Details" | "View Logs" so it's still possible to get a single tick's logs from the run it launched -- i think otherwise it'd take a click through to the sensor / schedule page, and then finding the tick in the tick history there.